### PR TITLE
add log & about

### PR DESCRIPTION
**Pull request by JeremyLi28**
@JavierJia add the about page, there are some parameters in it I don't know, such as the number of records. Can you add them?

### CODE REVIEW

File: src/main/resources/twitter-demo/tweetbook.tpl [hunk 1]
```diff
@@ -169,6 +182,19 @@
...
+                <p><b>Cloudberry</b> is a research prototype to support interactive analytics and visualization of large amounts of spatial-temporal data. </p>
+                <p> Basic Information: </p>
+                <ul>
+                    <li>Data set: Tweets</li>
+                    <li>Number of records: ???</li>
+                    <li>Collection period: [START TIME, END TIME]</li>
+                    <li>Total data size: ???</li>
+                    <li><a href="https://github.com/ISG-ICS/cloudberry">Source code</a></li>
+                </ul>
+                <p>The backend is running the big data management system Apache AsterixDB to support large compute clusters. For questions and comments, please contact <b>cloudberry@ics.uci.edu</b></p>
```
Comment (chenlica) [inline src/main/resources/twitter-demo/tweetbook.tpl]: Let's use Jianfeng's email address for now before the email list is approved.

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,7 @@
 target/
 .idea/
 *.iml
+
+logs/
+neo/
```
Comment (JavierJia) [inline .gitignore]: why ignore neo?

File: src/main/resources/twitter-demo/tweetbook.tpl [hunk 3]
```diff
@@ -169,6 +182,19 @@
                     </tbody>
                 </table>
             </div>
+            <div id="about" class="tab-pane active">
+                <h1> About </h1>
+                <p><b>Cloudberry</b> is a research prototype to support interactive analytics and visualization of large amounts of spatial-temporal data. </p>
+                <p> Basic Information: </p>
+                <ul>
+                    <li>Data set: Tweets</li>
+                    <li>Number of records: ???</li>
```
Comment (JavierJia) [inline src/main/resources/twitter-demo/tweetbook.tpl]: 18,953,055 records.

File: src/main/resources/twitter-demo/tweetbook.tpl [hunk 4]
```diff
@@ -169,6 +182,19 @@
...
+                    <li>Collection period: [START TIME, END TIME]</li>
```
Comment (JavierJia) [inline src/main/resources/twitter-demo/tweetbook.tpl]: From 2016-03-31 to 2016-04-07

File: src/main/resources/twitter-demo/tweetbook.tpl [hunk 5]
```diff
@@ -169,6 +182,19 @@
...
+                    <li>Total data size: ???</li>
```
Comment (JavierJia) [inline src/main/resources/twitter-demo/tweetbook.tpl]: 17G bytes.

**Comment by JavierJia** [comment]
@JeremyLi28 Added 👍

File: src/main/resources/twitter-demo/tweetbook.tpl (hunk 1 above)
Comment (JavierJia) [inline src/main/resources/twitter-demo/tweetbook.tpl]: My email is jianfenj@ics.uci.edu

**Comment by chenlica** [comment]
One more comment: The search box currently shows "Key words". Can we change it to some good suggested words such as "zika"?  Make sure that keyword is interested with precalculated results.  Also no "trump" please :-)

**Pull request merged**